docs/cue-lists/editing-cue-lists.md

```diff
@@ -59,36 +59,40 @@ to enter the new cue number - once you have changed the number the cue will move
 its correct position in the list.\
 To delete a cue, select the cue in the window and press \<Delete\>, followed by \<Delete\> again or \<Enter\> to confirm.
 
-You can also use a `command-line` style series of keypresses.
+You can also use a command-line style series of keypresses.
 
-**Copy/Move within the same playback:**
+- **{fader select}** is a playback fader select button, **n** is the cue number and
+  sections in square brackets are optional). The \<@\> button is the one by the numeric keypad.
+  
+- In all these commands you can **move** cues rather than **copying** by pressing \<Move\> instead of \<Copy\>.
 
-`COPY/MOVE <playback> <cue> [THRO <cue>] [NOT <cue>] [AND <cue>] @ <cue> ENTER`
 
-`COPY/MOVE <playback> <cue> [THRO <cue>] [NOT <cue>][AND <cue>] ENTER <cue> ENTER`
+**Copy cue within the same playback:**
 
-**Copy/Move to the end of the same playback:**
+\<Copy\> {fader select} n [\<Thru\> n] [\[Not\] n] [\<And\> n] \<@\> n \<Enter\>
```

Comment:
This is far clearer and looks really good:

---

![image](https://user-images.githubusercontent.com/2584408/94292277-a479c180-ff54-11ea-8275-1608a3373c5b.png)

---

A couple of things:
1. Should we list it without the optional steps first to make it clearer as this is what many users will be looking for
2. It may be clearer with **n** being bold to match the description above

Reply:
Have changed to table format to match other sections of manual.
Optional steps using THRU etc moved to a note+example below the table

Reply:
Looks really good